hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCpConstants.java

```diff
@@ -142,6 +142,13 @@ private DistCpConstants() {
       "distcp.blocks.per.chunk";
 
   public static final String CONF_LABEL_USE_ITERATOR = "distcp.use.iterator";
+
+  /** Distcp -update to use modification time of source and target file to
+   * check while skipping.
+   */
+  public static final String CONF_LABEL_UPDATE_MOD_TIME =
```

Comment:
going to need docs.

Reply:
Not sure if you mean ".md" file modification or the modification to already added java docs? I'll add some more info to the javadocs about the constant.

Reply:
1. javadoc to include {@value}
2. distcp docs to explain the option